### PR TITLE
110-rotation

### LINKED ISSUE #110: Port spatial rotation transform
a subtask of #62, port a spatial rotation transform into Monai

e.g.:
https://simpleitk-prototype.readthedocs.io/en/latest/user_guide/transforms/plot_transforms.html#rotation

### PR DESCRIPTION
**Pull request by madil90**
Fixes #110  .

### Description
Add the rotation transform. Tests coming soon.

### Status
**Work in progress**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### CODE REVIEW

**Comment by madil90** [comment]
@wyli @ericspod Same behavior as https://github.com/Project-MONAI/MONAI/pull/125#issuecomment-592270493 replicated here. Resolution of that comment will be applied.

File: monai/transforms/transforms.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 import torch
 import numpy as np
+from skimage.transform import rotate
```
Comment (ericspod) [inline monai/transforms/transforms.py]: skimage isn't listed as a dependency in requirements.txt, an import won't be possible if it's missing.

Comment (tvercaut) [reply monai/transforms/transforms.py]: Also, as briefly discussed in https://github.com/Project-MONAI/MONAI/issues/112#issuecomment-590517152, pytorch provides a handy `grid_sample` function which seems also much more efficient than skimage.transform (at least for generic warps in 2D). See https://colab.research.google.com/drive/17pI6xPGLTGcpGBulBrWZeNnBfLAwzqZT

Comment (wyli) [reply monai/transforms/transforms.py]: strangely the CI test passed without modifying `requirements.txt`, @IsaacYangSLA looks like the CI docker image has common packages preinstalled, could we have a minimal docker image instead?

Comment (madil90) [reply monai/transforms/transforms.py]: Updated to use scipy.ndimage. Scikit-image doesn't work for 3D images properly. Also added to requirements and test.

Comment (madil90) [reply monai/transforms/transforms.py]: @tvercaut That request is being handled in another PR.

**Pull request merged**